### PR TITLE
Added support for more Veyron boards

### PR DESCRIPTION
**Pull request by colemancda**
Tested on Jerry (CTL Chomebook J4)

### CODE REVIEW

**Comment by colemancda** [comment]
Latest commit fixes WiFi support on CTL Chromebook J4/J2

File: baseboard/veyron/kernel-config [hunk 1]
```diff
@@ -1,15 +1,15 @@
 #
```
Comment (Maccraft123) [inline baseboard/veyron/kernel-config:1]: Uhh, is there any reason why diff of this file is THAT big?

**Review by Maccraft123 — CHANGES_REQUESTED** [review]
Aside from what I already addressed this is ready to merge after testing that it builds.
Is hardware support as full as other veyron boards?

File: baseboard/veyron/kernel-config (hunk 1 above)
Comment (colemancda) [reply baseboard/veyron/kernel-config:1]: Due to using `make menuconfig` with kernel `5.14.0-rc6`

Comment (colemancda) [reply baseboard/veyron/kernel-config:1]: I tried manually editing at first but there are some implicit config settings needed to get the Marvell WiFi working.

**Pull request closed without merging**